apps/berik-inntektsmelding-service/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/berikinntektsmeldingservice/BerikInntektsmeldingService.kt

```diff
@@ -44,7 +45,7 @@ data class Steg0(
     val inntektsmeldingId: UUID,
     val skjema: SkjemaInntektsmelding,
     val innsending: Innsending?, // TODO: Kan dele opp API-innsending-berik i egen service
-    val innsendingId: Long,
+    val innsendingId: Long?,
```

Comment:
`innsendingId` gjøres nullable flere steder for å takle at den ikke kommer i fremtiden, da den vil fjernes etter en overgangsfase.